persistence/src/commonMain/db_user/migrations/27.sqm

```diff
@@ -136,42 +110,11 @@ LEFT JOIN MessageUnknownContent AS UnknownContent ON Message.id = UnknownContent
-LEFT JOIN MessageQuoteDetails AS QuotedMessage ON Message.id = QuotedMessage.messageId AND Message.conversation_id = QuotedMessage.conversationId
-LEFT JOIN MessageNewConversationReceiptModeContent AS NewConversationReceiptMode ON Message.id = NewConversationReceiptMode.message_id AND Message.conversation_id = NewConversationReceiptMode.conversation_id
-LEFT JOIN MessageConversationReceiptModeChangedContent AS ConversationReceiptModeChanged ON Message.id = ConversationReceiptModeChanged.message_id AND Message.conversation_id = ConversationReceiptModeChanged.conversation_id
-LEFT JOIN SelfUser;
 
-CREATE VIEW IF NOT EXISTS MessagePreview
-AS SELECT
-    Message.id AS id,
-    Message.conversation_id AS conversationId,
-    Message.content_type AS contentType,
-    Message.creation_date AS date,
-    Message.visibility AS visibility,
-    Message.sender_user_id AS senderUserId,
-    User.name AS senderName,
-    User.connection_status AS senderConnectionStatus,
-    User.deleted AS senderIsDeleted,
-    SelfUser.id AS selfUserId,
-    (Message.sender_user_id == SelfUser.id) AS isSelfMessage,
-    MemberChangeContent.member_change_list AS memberChangeList,
-    MemberChangeContent.member_change_type AS memberChangeType,
-    ConversationNameChangedContent.conversation_name AS updateConversationName,
-    Conversation.name AS conversationName,
-    (Mention.user_id IS NOT NULL) AS isMentioningSelfUser,
-    TextContent.is_quoting_self AS isQuotingSelfUser,
-    TextContent.text_body AS text,
-    AssetContent.asset_mime_type AS assetMimeType,
-    (Message.creation_date > Conversation.last_read_date) AS isUnread,
-    IFNULL((Message.creation_date > IFNULL(Conversation.last_notified_date, 0)), FALSE) AS shouldNotify,
-    Conversation.muted_status AS mutedStatus,
-    Conversation.type AS conversationType
-FROM Message
-LEFT JOIN SelfUser
-LEFT JOIN User ON Message.sender_user_id = User.qualified_id
-LEFT JOIN Conversation AS Conversation ON Message.conversation_id == Conversation.qualified_id
-LEFT JOIN MessageMemberChangeContent AS MemberChangeContent ON Message.id = MemberChangeContent.message_id AND Message.conversation_id = MemberChangeContent.conversation_id
-LEFT JOIN MessageMention AS Mention ON Message.id == Mention.message_id AND SelfUser.id == Mention.user_id
-LEFT JOIN MessageConversationChangedContent AS ConversationNameChangedContent ON Message.id = ConversationNameChangedContent.message_id AND Message.conversation_id = ConversationNameChangedContent.conversation_id
-LEFT JOIN MessageAssetContent AS AssetContent ON Message.id = AssetContent.message_id AND Message.conversation_id = AssetContent.conversation_id
-LEFT JOIN MessageTextContent AS TextContent ON Message.id = TextContent.message_id AND Message.conversation_id = TextContent.conversation_id;
+-- joins for quoted messages
+LEFT JOIN Message AS QuotedMessage ON QuotedMessage.id = TextContent.quoted_message_id AND QuotedMessage.conversation_id = TextContent.conversation_id
```

Comment:
I'm not an expert with changing some code in already created sqm file but have you tested it with current updating it from current version in store?

Reply:
i will give it a double check with by installing current RC and then updating to this 👍🏼

Reply:
did some testing and it works just fine + all unit tests are passing 🚀